src/solvers/smt1/smt1_conv.cpp

```diff
@@ -2765,12 +2763,10 @@ void smt1_convt::set_to(const exprt &expr, bool value)
 
   #if 0
   out << "; CONV: "
-                << from_expr(expr) << "\n";
+                << expr.as_string() << '\n';
   #endif
 
-  out << ":assumption ; set_to "
-      << (value?"true":"false") << "\n"
-      << " ";
+  out << ":assumption ; set_to " << (value ? "true" : "false") << '\n' << " ";
```

Comment:
These changes seem strange/unrelated? Why not `format` in the above?

Reply:
That was clang-format

Reply:
I suppose that's because of the `"\n"` -> `'\n'`? Just doesn't really seem related to this PR?